Doc/Tutorial/chapter_pairwise.rst

```diff
@@ -320,6 +321,30 @@ alignments if segments with a score 0 can be added to the alignment. We
 follow the suggestion by Waterman & Eggert
 [Waterman1987]_ and disallow such extensions.
 
+If `aligner.mode` is set to `"fogsaa"`, then the Fast Optimal Global Alignment
+Algorithm [Chakraborty2013]_ with some modifications is used. This mode
+calculates a global alignment, but it is not like the regular `"global"` mode.
+It is best suited for long alignments between similar sequences. If the the
+match score is less than the mismatch score or any gap score, or if any gap
+score is greater than the mismatch score, then a warning is raised and the
+algorithm may return incorrect results. Unlike other modes that may return more
+than one alignment, FOGSAA always returns only one alignment.
```

Comment:
Key point to include here is that FOGSAA relies on heuristic, and (unlike the other alignment algorithms) is not guaranteed to find the alignment with the highest score.

Reply:
I changed the description to explain how the heuristic is used. FOGSAA should be guaranteed to find the optimal alignment (one of the alignments returned by Needleman-Wunsch) if the assumptions of the heuristic are not violated. These are what the warnings are for: if a warning is raised then the heuristic *may* be incorrect.

Does the new explanation make this clear to users, or should I modify it?

Reply:
I think it's fine.  Users who want to know the details can look in the literature.